server/neptune/workflows/internal/temporal/duration.go

```diff
@@ -0,0 +1,36 @@
+package temporal
+
+import (
+	"time"
+
+	"go.temporal.io/sdk/workflow"
+)
+
+type NextDay func(time.Time) time.Time
+
+func NextBusinessDay(d time.Time) time.Time {
+	d = d.Add(24 * time.Hour)
+
+	for d.Weekday() == time.Saturday || d.Weekday() == time.Sunday {
+		d = d.Add(24 * time.Hour)
```

Comment:
should you add 2 days for saturday?

Reply:
This is a for loop, it'll add days until monday

Reply:
oops missed the loop! lgtm